front/src/modules/pathfinding/components/Itinerary/DisplayItinerary/Vias.tsx

```diff
@@ -16,9 +17,10 @@ import ViaStopDurationSelector from './ViaStopDurationSelector';
 type ViasProps = {
   shouldManageStopDuration?: boolean;
   zoomToFeaturePoint: (lngLat?: Position, id?: string) => void;
+  isInvalid: (step: PathStep | null) => boolean;
```

Comment:
Since this function is in an utils, you can remove this props and directly use the function

Reply:
done